.github/actions/terraform/main.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Path to location of terraform files
+TERRAFORM_DIR=${INPUT_TERRAFORM_DIR:-${TERRAFORM_DIR}}
```

Comment:
Github Actions transform the [action input `terraform_dir`](https://github.com/nervosnetwork/ckb/pull/2679/files#diff-60065d341a47aaac2c5285ccaf4bc533609632cf74aa7603f80b1de0f2e37ac2R6) into environment variable `INPUT_TERRAFORM_DIR`.